cg/cli/demultiplex/sample_sheet.py

```diff
@@ -1,19 +1,34 @@
 import logging
 from pathlib import Path
-from typing import List
+from typing import List, Union
 
 import click
+from pydantic import ValidationError
+
 from cg.apps.demultiplex.demultiplex_api import DemultiplexingAPI
 from cg.apps.demultiplex.sample_sheet.create import create_sample_sheet
-from cg.apps.lims.samplesheet import LimsFlowcellSample, flowcell_samples
+from cg.apps.lims.samplesheet import (
+    LimsFlowcellSample,
+    LimsFlowcellSampleBcl2Fastq,
+    LimsFlowcellSampleDragen,
+    flowcell_samples,
+)
 from cg.exc import FlowcellError
 from cg.models.cg_config import CGConfig
 from cg.models.demultiplex.flowcell import Flowcell
 from cgmodels.demultiplex.sample_sheet import get_sample_sheet_from_file
-from pydantic import ValidationError
 
 LOG = logging.getLogger(__name__)
 
+OPTION_BCL_CONVERTER = click.option(
```

Comment:
I think you are reusing the same option several time. It could be a demultiplex constant that you can import into the different CLI commands. See the MIP workflow for an example